test/int/integration_suite_test.go

```diff
@@ -112,57 +100,66 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		fmt.Println("Skipping int SynchronizedBeforeSuite, AKO_INT_TEST is not set")
 		return nil
 	}
-	By("bootstrapping test environment")
-
-	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "..", "config", "crd", "bases")},
-	}
 
-	cfg, err := testEnv.Start()
-	Expect(err).ToNot(HaveOccurred())
-	Expect(cfg).ToNot(BeNil())
+	By("Validating configuration data is available", func() {
+		Expect(os.Getenv("ATLAS_ORG_ID")).ToNot(BeEmpty())
+		Expect(os.Getenv("ATLAS_PUBLIC_KEY")).ToNot(BeEmpty())
+		Expect(os.Getenv("ATLAS_PRIVATE_KEY")).ToNot(BeEmpty())
+	})
 
 	var b bytes.Buffer
-	e := gob.NewEncoder(&b)
-	err = e.Encode(*cfg)
-	Expect(err).NotTo(HaveOccurred())
 
-	fmt.Printf("Api Server is listening on %s\n", cfg.Host)
+	By("Bootstrapping test environment", func() {
+		useExistingCluster := os.Getenv("USE_EXISTING_CLUSTER") != ""
+		testEnv = &envtest.Environment{
+			CRDDirectoryPaths:  []string{filepath.Join("..", "..", "config", "crd", "bases")},
+			UseExistingCluster: &useExistingCluster,
+		}
+
+		cfg, err := testEnv.Start()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(cfg).ToNot(BeNil())
+
+		e := gob.NewEncoder(&b)
+		err = e.Encode(*cfg)
+		Expect(err).ToNot(HaveOccurred())
+
+		fmt.Printf("Api Server is listening on %s\n", cfg.Host)
```

Comment:
let's use the intrinsic logger of the test framework instead of `fmt.Println`/`fmt.Printf`